_bibliography/external/tool_pandt_project.bib

```diff
@@ -0,0 +1,146 @@
+%% BibTeX errors.
+%
+%
+% The entries in this file are sorted by YEAR (ascending).
+% If inserting new entries, please put them accordingly.
+%
+%
+% ALSO NOTE:
+%
+% Please, stick to the following formatting convention to easy maintainability:
+%
+% - indent keys of entries with 2 spaces (NOT tabs!)
+%
+% - order the keys alphabetical
+%
+% - enclose all values within curly brakets '{...}'
+%
+% - do line breaks at 100 chars; this is ................................................. here -->|
+%
+% - and add a continuation indent of 2 after line breaks
+%
+% - put a colon ',' as well after the last key/value pair of an entry
+%
+%
+% Please, treat this file not different from usual source code files.
+%
+% Thus, stick to the concisely used formatting pattern with tab indention, lower cased keywords,
+% curly brackets for value enclosure and alphabetical sorted keywords within one BibTeX entry.
+%
+%
+
+@inproceedings{FringsEtAl2009,
+  acmid = {1654077},
+  articleno = {17},
+  author = {Frings, Wolfgang and Wolf, Felix and Petkov, Ventsislav},
+  booktitle = {Proceedings of the Conference on High Performance Computing Networking, Storage and 
+    Analysis},
+  doi = {10.1145/1654059.1654077},
+  isbn = {978-1-60558-744-8},
+  location = {Portland, Oregon},
+  numpages = {11},
+  pages = {17:1--17:11},
+  publisher = {ACM},
+  series = {SC '09},
+  title = {Scalable Massively Parallel I/O to Task-local Files},
+  url = {http://doi.acm.org/10.1145/1654059.1654077},
+  year = {2009},
+} 
+
+@article{GeimerEtAl2010,
+  author = {Geimer, Markus and Wolf, Felix and Wylie, Brian J. N. and Abraham, Erika and
+    Becker, Daniel and Mohr, Bernd},
+  acmid = {1753234},
+  address = {Chichester, UK},
+  doi = {10.1002/cpe.v22:6},
+  issn = {1532-0626},
+  issue_date = {April 2010},
+  journal = {Concurr. Comput. : Pract. Exper.},
+  keywords = {parallel computing, performance analysis, scalability},
+  month = {apr},
+  number = {6},
+  numpages = {18},
+  pages = {702--719},
+  publisher = {John Wiley and Sons Ltd.},
+  title = {The Scalasca Performance Toolset Architecture},
+  url = {http://dx.doi.org/10.1002/cpe.v22:6},
+  volume = {22},
+  year = {2010},
+}  
+
+
+
+@inproceedings{KnuepferEtAl2012,
+  author       = {Knüpfer, A. and Rössel, C. and an Mey, D. and Biersdorff,
+    S. and Diethelm, K. and Eschweiler, D. and Geimer, M. and Gerndt, M. and Lorenz, D.
+    and Malony, A.D. and Nagel, W.E. and Oleynik, Y. and Philippen, P. and Saviankou, P.
+    and Schmidl, D. and Shende, S.S. and Tschüter, R. and Wagner, M. and Wesarg, B. and Wolf, F.},
+  booktitle     = {Tools for High Performance Computing 2011, Proceedings of the 5th
+    International Workshop on Parallel Tools for High Performance Computing, September 2011, Dresden, 
+    ed.: H. r Brunst, M.S. Müller, W.E. Nage, M.M. Resch, Springer, 2012. - 978-3-642-31475-9. -
+    S. 79 - 91},
+  cin          = {JSC},
```

Comment:
I know I'm nitpicking, but do you mind sticking to one single formatting style for BibTeX files?